src/common/motech-file-upload/motech-file-upload.html

```diff
@@ -7,4 +7,6 @@
     <span class="fileinput-exists">{{'server.button.changeFile' | translate}}</span>
     <input type="file" id="fileInput" name="file" class="input-file"/>
 </span>
-<button class="button-delete fileinput-exists" data-dismiss="fileinput">{{'server.button.remove' | translate}}</button>
\ No newline at end of file
+<div>
```

Comment:
@tomaszzawadzki Instead of adding a `<div>` element, please change the width of the `<div class="button-file">` so that both buttons are next to each other and fit within the size of the other form elements.